modules/core/src/lib/attribute-transition-utils.js

```diff
@@ -107,3 +108,10 @@ export function padBuffer({
 
   fromState.setData({data});
 }
+
+export function getBufferLength(buffer) {
+  if (buffer.data) {
```

Comment:
Is this temporary? wouldn't this always be false once luma PR lands?

Reply:
Doesn't it keep some data for debug purpose? If not, can we store the element length in the Buffer object?

Reply:
Yes we will be adding getLength support to Buffer eventually, it is going to be part of this PR : https://github.com/uber/luma.gl/pull/423

Reply:
if all you need is 'BYTES_PER_ELEMENT', can you use `getTypedArrayFromGLType` + buffer.accessor.type ? We can clean this up once we have full support in luma.gl

Reply:
Added support in luma.gl https://github.com/uber/luma.gl/pull/592